c2cgeoportal/scaffolds/__init__.py

```diff
@@ -125,25 +167,33 @@ def _set_mobile_title_in_vars(self, command, vars):
 
         vars["mobile_application_title"] = mobile_title
 
-    def _set_srid_in_vars(self, command, vars):
+    def _set_extent_in_vars(self, command, vars):
         """
-        Set the SRID into the vars dict.
+        Set the Extent into the vars dict.
         """
-        srid = None
+        extent = None
         for arg in command.args:
-            m = re.match("srid=(\d+)", arg)
+            m = re.match("extent=(\d+, ){3}(\d+ )", arg)
             if m:
-                srid = m.group(1)
+                extent = m.group(1)
                 break
-        if srid is None:
-            prompt = "Spatial Reference System Identifier " \
-                     "(e.g. 21781): "
-            srid = input_(prompt).strip()
-        try:
-            vars["srid"] = int(srid)
-        except ValueError:
-            raise ValueError(
-                "Specified SRID is not an integer")
+        if extent is None:
+            extent = self._epsg2bbox(vars["srid"])
+        if extent is None:
+            prompt = "Extent (minx, miny, maxx, maxy): " \
+                    "in EPSG:"+vars["srid"]+" projection: "
+            extent = input_(prompt).strip()
+        vars["extent_mapserver"] = " ".join(extent)
+        vars["extent_viewer"] = json.dumps(extent)
+
+    def _epsg2bbox(self, srid):
+        r = requests.get('http://epsg.io/?format=json&q='+str(srid))
+        bbox = r.json()['results'][0]['bbox']
+        r = requests.get('http://epsg.io/trans?s_srs=4326&t_srs='+str(srid)+'&data='+str(bbox[1])+','+str(bbox[0]))
```

Comment:
it's better to use python template (easer to understand) =>
`'http://epsg.io/trans?s_srs=4326&t_srs=%i&data=%i,%i' % (srid, bbox[1], bbox[0])